.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Report any bugs or unexpected behaviour
+title: 'bug: '
+labels: bug, track:maintenance
+assignees: ''
+
+---
+
+### Problem
+A clear and concise description of what the bug is.
+
+### Impact
+Indicate how significant you believe the impact of the bug is. Bugs that lead to data loss or corruption would be considered `critical`. In such cases, please also add the `critical` label.
+
+### To reproduce
+If you can reproduce the behaviour, steps to reproduce:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+### Expected behaviour
+A clear and concise description of what you expected to happen.
+
+### Screenshots/logs
+If applicable, add screenshots or logs to help explain your problem.
+
+### services version/commit hash
+State the version of services where you've encountered the bug or, if built off a specific commit, the relevant commit hash.
+- e.g. `v0.9` or `ed53bcd`
```

Comment:
We could tell people to use the endpoint `/v1/version` to find out which version the backend was running when encountering the bug. Otherwise they might have a hard time figuring that out. However, this would probably also require us to tell them to use `barn.api.cow.fi..` when on staging and `api.cow.fi...` on prod.

Reply:
Great suggestion, will include this.